### DISCUSSION TITLE
configuration-and-or-dependency-management

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#13] [Technical Blog]
# configuration-and-or-dependency-management

https://wsbctechnicalblog.github.io/configuration-and-or-dependency-management.html

**Comment by grabnerandi** [comment]
I think modern observability & security tools can help you. Observability gives you all dependencies between your hosts, processes and services and those tools also automatically tell you what libriries and runtimes are loaded and whether they have vulnerabilities. 
Without making this a commercial for a product I still want to highlight that we at Dynatrace provide this in our platform. 
Now. The way to solve your problem is to use such a solution across all your stages so that you automatically validate whether updates really have the intended impact BEFORE you promote to prod. 
Our users are therefore integrating Dynatrace into their delivery pipelines to act as Quality Gates as well as to automate Release and Deployment Validation
Happy to chat more

Fyi. Here a link with some more information on dynatrace appsec: https://www.dynatrace.com/news/blog/dynatrace-extends-appsec-to-go/

**Reply by wpschaub** [reply]
Thank you Andi, this is exciting news! I will discuss this with our engineers and revert.
